packages/beat/manifest.yml

```
@@ -0,0 +1,35 @@
+name: beat
+title: Beat
+version: 0.0.1
+description: Beat Integration
+type: integration
+release: experimental
+conditions:
+  kibana:
+    version: ^8.7.0
```

Try this to solve the issue with CI:
```suggestion
  kibana.version: ^8.7.0
```